src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -107,4 +156,31 @@ describe('<TranscriptsListItemInfo /', () => {
     userEvent.click(proteinLink);
     expect(defaultProps.onProteinLinkClick).toHaveBeenCalled();
   });
+
+  it('display metadata when it is available', () => {
+    const { queryByText } = renderComponent({
+      transcript: createGencodeBasicTranscript(),
+      expandMoreInfo: true
+    });
+    const refseqText = queryByText('gencode basic');
```

Comment:
refSeq refers to a very particular kind of data: the NCBI reference sequence that is part of mane metadata. Gencode basic is not refSeq

Reply:
Sorry, copy paste issue :-)